packages/vscode-extension/src/commonlib/codeFlowLogin.ts

```diff
@@ -216,6 +228,54 @@ export class CodeFlowLogin {
     return accessToken;
   }
 
+  async loginInCodeSpace(scopes: Array<string>): Promise<string> {
+    let callbackUri: Uri = await env.asExternalUri(Uri.parse(`${env.uriScheme}://TeamsDevApp.ms-teams-vscode-extension/auth-complete`));
```

Comment:
Any existing constant defined for `TeamsDevApp.ms-teams-vscode-extension`?

Reply:
Add a constant definition for this.